docs/integrations/prometheus.md

```diff
@@ -29,6 +29,9 @@ metrics:
   sanitization: prometheus
   samplingRate: 1.0
   extended: none
+  
+  tagOptions:
+  idScheme: quoted
```

Comment:
This should be under `tagOptions`.

Reply:
Thank you, I corrected it in a new commit.